apps/api/src/app/analytics/analytics.controller.ts

```diff
@@ -0,0 +1,18 @@
+import { Body, Controller, Post } from '@nestjs/common';
+import { AnalyticsService } from '@novu/application-generic';
+
+@Controller({
+  path: 'analytics',
+})
+export class AnalyticsController {
```

Comment:
Once we add the Auth guard, we also need to use the `@SkipThrottle` decorator to ignore these endpoints from consuming the rate limiting quota for the User's Organization.